### PR TITLE
add fr version

### PR DESCRIPTION
**Pull request by LeTengu**

### CODE REVIEW

File: dialogTemplate/fr.json [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+				"je veux jouer avec toi",
+				"je voudrais jouer avec toi",
+				"jouons ensemble",
+				"joue avec moi",
+				"joue avec moi s'il te plaît",
+				"peux-tu s'il te plaît {flip a coin:=>WhichGame}",
+				"peux-tu {flip a coin:=>WhichGame}",
+				"peux-tu {roll a dice:=>WhichGame}",
+				"jouons à {scissors rock paper:=>WhichGame}",
+				"jouer à {scissors rock paper:=>WhichGame}"
```
Comment (maxbachmann) [inline dialogTemplate/fr.json]: the slots should be replaced with the french words

File: dialogTemplate/fr.json [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+		},
+		{
+			"name": "AnswerHeadsOrTail",
+			"description": "L'utilisateur répond pile ou face",
+			"enabledByDefault": false,
+			"utterances": [
+				"{Heads:=>HeadsOrTails}",
+				"je choisis {tail:=>HeadsOrTails}",
+				"je prends {tail:=>HeadsOrTails}",
+				"{ship:=>HeadsOrTails} s'il te plaît"
```
Comment (maxbachmann) [inline dialogTemplate/fr.json]: The slots should be replaced with the french version

File: dialogTemplate/fr.json [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+		},
+		{
+			"name": "AnswerRockPaperOrScissors",
+			"description": "L'utilisateur répond pierre, papier ou ciseaux",
+			"enabledByDefault": false,
+			"utterances": [
+				"{rock:=>RockPaperOrScissors}",
+				"je choisis {paper:=>RockPaperOrScissors}",
+				"je prends {scissors:=>RockPaperOrScissors}",
+				"{stone:=>RockPaperOrScissors} s'il te plaît"
```
Comment (maxbachmann) [inline dialogTemplate/fr.json]: The slots should be replaced with the french version

**Review by maxbachmann — CHANGES_REQUESTED** [review]

**Comment by maxbachmann** [comment]
Thanks for even more translations :)
some slots are still using the english. When they are replaced with the french counterpart this can be merged

File: dialogTemplate/fr.json (hunk 1 above)
Comment (LeTengu) [reply dialogTemplate/fr.json]: Ok, but here for example, WhichGame can't change(Slot Name)… True ?
So I must do: "{lancer une pièce:=>WhichGame}" ?
Thanks you.

Comment (maxbachmann) [reply dialogTemplate/fr.json]: yes the slot names should always stay the same as in english. The same is true for the first slot value in many cases. (the way you did it)

**Pull request merged**